src/euphorie/client/tests/test_consultancy.py

```diff
@@ -202,24 +225,83 @@ def test_validate_risk_assessment(self):
                 view.request.method = "POST"
                 view.request.form = {"approved": "1"}
                 view()
-                # consultant stays set
-                self.assertEqual(
-                    self.traversed_session.session.consultancy.account,
-                    self.consultant,
-                )
+
+        # consultant stays set
+        self.assertEqual(
+            self.traversed_session.session.consultancy.account,
+            self.consultant,
+        )
+        self.assertEqual(self.traversed_session.session.consultancy.status, "validated")
+        with api.env.adopt_user(user=self.owner):
+            with self._get_view(
+                "consultancy",
+                self.traversed_session,
+                self.traversed_session.session,
+            ) as view:
                 self.assertEqual(
-                    self.traversed_session.session.consultancy.status, "validated"
+                    view.validated_info.consultant_email,
+                    "michel.moulin@example-consultancy.com",
                 )
+                self.assertEqual(view.validated_info.consultant_name, "Michel Moulin")
+            self.assertLessEqual(
+                abs(
+                    view.validated_info.raw_time
+                    - datetime.utcnow().replace(tzinfo=timezone.utc)
+                ),
+                timedelta(seconds=5),
+            )
 
-                # TODO check that assessment is locked
-                # self.assertTrue(self.traversed_session.session.locked)
+        # TODO check that assessment is locked
+        # self.assertTrue(self.traversed_session.session.locked)
 
-                self.assertEqual(len(mail_fixture.storage), 2)
-                recipients = {
-                    mail_fixture.storage[0][0][1],
-                    mail_fixture.storage[1][0][1],
+        self.assertEqual(len(mail_fixture.storage), 2)
+        recipients = {
+            mail_fixture.storage[0][0][1],
+            mail_fixture.storage[1][0][1],
+        }
+        self.assertSetEqual(
+            recipients,
+            {"valerie@labyrinth.social", "jessica@labyrinth.social"},
+        )
+
+    def test_delete_consultant(self):
+        now = datetime.utcnow().replace(tzinfo=timezone.utc)
+        self.traversed_session.session.consultancy = Consultancy(
```

Comment:
It seems to me we should not have this column but just rely on the validation event.

Reply:
I'm not sure which column you mean but see https://github.com/euphorie/Euphorie/pull/546#issuecomment-1532642945 for why I don't want to rely on the event alone.